kafka-ui-react-app/.eslintrc.json

```diff
@@ -29,7 +29,8 @@
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "jsx-a11y/label-has-associated-control": "off",
     "import/prefer-default-export": "off",
-    "@typescript-eslint/no-explicit-any": "error"
+    "@typescript-eslint/no-explicit-any": "error",
+    "no-console": "off"
```

Comment:
There is no need to disable `no-console` rule for the whole project. We can disable it for `serviceWorker.ts` file.